### PR TITLE
Rough working version of selector generator

### PR DESCRIPTION
**Pull request by calebjclark**
This PR integrates the Selector Generator with the built-in Devtools Element Selector. 

It also extends the right-click contextMenu inside the Devtools Elements Panel (right click on any element to add/remove from Selector Generator).

The actual Selector Generator is missing everything except Single Element selectors. In order to push a working version I had to rollback all of my Multiple Elements selector code. This also rolled back the ability to remove selector parts (i.e., cutting an ID from selectors).

Finally, there is zero integration with Multiverses.

### CODE REVIEW

File: apps/chromealive-core/lib/bridges/BridgeToDevtoolsPrivate.ts [hunk 1]
```diff
@@ -39,27 +39,40 @@ export default class BridgeToDevtoolsPrivate extends EventEmitter {
...
+          (${interceptInspectElementMode.toString()})('${___sendToCore}', '${MessageEventType.InspectElementModeChanged}');
+          (${interceptElementPanelOnHighlight.toString()})('${___sendToCore}', '${MessageEventType.HideElementOptionsOverlay}');
+          (${interceptElementPanelOnRemoveHighlight.toString()})('${___sendToCore}', '${MessageEventType.RemoveHideFromElementOptionsOverlay}');
+          (${injectContextMenu.toString()})('${___sendToCore}', '${MessageEventType.UpdateElementOptions}');
         })();`,
       }),
       this.getDevtoolsTabId.bind(this, devtoolsSession),
       devtoolsSession.send('Runtime.runIfWaitingForDebugger'),
     ]).catch(() => null);
+    // (${interceptElementOverlayDispatches.toString()})('${___sendToCore}', '${MessageEventType.CloseElementOptionsOverlay}');
```
Comment (blakebyrnes) [inline apps/chromealive-core/lib/bridges/BridgeToDevtoolsPrivate.ts:75]: meant to be commented out?

File: apps/chromealive-extension/package.json [hunk 2]
```diff
@@ -33,9 +36,13 @@
     "@vue/compiler-sfc": "^3.0.0",
     "autoprefixer": "^9",
     "babel-loader": "^8.2.2",
+    "chrome-api-mock": "^0.0.2",
     "copy-webpack-plugin": "^9.0.1",
     "glob": "^7.1.6",
+    "jest-chrome": "^0.7.2",
     "postcss": "^7",
+    "sass": "^1.43.4",
+    "sass-loader": "sass-loader@10.1.1",
```
Comment (blakebyrnes) [inline apps/chromealive-extension/package.json:45]: This seems to be breaking github actions

File: apps/chromealive-core/lib/bridges/BridgeToDevtoolsPrivate.ts (hunk 1 above)
Comment (calebjclark) [reply apps/chromealive-core/lib/bridges/BridgeToDevtoolsPrivate.ts:75]: Yes

File: apps/chromealive-extension/package.json (hunk 2 above)
Comment (calebjclark) [reply apps/chromealive-extension/package.json]: Fixed

Comment (calebjclark) [reply apps/chromealive-extension/package.json]: Fixed

**Pull request merged**